_layouts/video-card.html

```diff
@@ -26,6 +61,38 @@
                   <a  class="text-black ogp-card__link--unset" href="{{ site.baseurl }}{{single_video.directory}}{{single_video.page_name}}">
                       <b>{{single_video.title}} </b>
                   </a>
+                  <button type="button" class="usa-button usa-button--outline" style="float:right;" id="btnShare">Share</button>
+                  <ul class="usa-button-group usa-button-group--segmented" style="float:right;display: none;" id="shareLinkUl">
+                    <li class="usa-button-group__item">
```

Comment:
please remove style and use custom class or uswds utilities classes

Reply:
used uswds class